pkgs/tools/misc/cp-p/default.nix

```
@@ -0,0 +1,31 @@
+{ lib
+, stdenv
+, fetchFromGitHub }:
+
+stdenv.mkDerivation {
+  name = "cp-p";
+  pname = "cp-p";
+
+  src = fetchFromGitHub {
+    owner = "Naheel-Azawy";
+    repo = "cp-p";
+    rev = "2e97ba534a5892c47a0317a038b19bcda221e5e6";
+    sha256 = "1ki8bp33d9xf4jinr0isvkxq3i86xi1kkv2f5x54as6i0yz9w7iq";
+  };
+  
+  # this cancels automatic make
+  dontBuild = true;
+  
+  installPhase = ''
+    mkdir -p $out/bin
+    cp cp-p mv-p $out/bin/
+  '';
```

```suggestion
  installPhase = ''
    runHook preInstall
    mkdir -p $out/bin
    cp cp-p mv-p $out/bin/
    runHook preInstall
  '';
```

That said, it's probably better to use the bundled Makefile and set `makeFlags` such that it does the correct thing.

---

If such a thing can be achieved then that would be nice

---

mkdir+cp can generally be combined into `install`:
```nix
    install -Dm555 cp-p mv-p -t $out/bin
```